vyxal/main.py

```diff
@@ -143,7 +143,8 @@ def execute_vyxal(file_name, flags, inputs, output_var=None, online_mode=False):
 
     ctx.stacks.append(stack)
     try:
-        exec(code)
+        locals()["stack"] = stack
+        exec(code, locals() | globals())
```

Comment:
Merging these creates a new object, so any changes to the new object don't affect the old `locals()` or `globals()`. (i.e., any assignments inside `code` won't affect any variables in `execute_vyxal()`). Is this what you want?

If not, use a `collections.chainmap`.

Reply:
I'm fine with it not actually changing locals or global. In fact, that's probably better in the bigger picture.